tests/test_wave_05.py

```diff
@@ -52,13 +52,18 @@ def test_new_genre_rec_from_empty_friends():
             }
         ]
     }
-
-    raise Exception("Test needs to be completed.")
+    # raise Exception("Test needs to be completed.")
```

Comment:
This test still has a `skip` on it and when I ran it, the test failed. I'll leave more detailed comments in Wave 5 in party.py